### DISCUSSION TITLE
Add option to specify CT/VM ID during install

### DISCUSSION THREAD

**Discussion by almostserious** [tteck/Proxmox#149] [Ideas]
Thanks a lot for these scripts. They make deploy for some of the services an absolute beeze.
I currently would like to have specific CTID for my containers though. 
So instead of relying on the script to just take the next available I am changing the scipt by hand to add my desired ID.

Would it be possible to add a selection like for the other options to either specify your own ID, or to just pick the next available?

**Comment by tteck** [comment]
That is exactly one of the configurable options in the upcoming v3 script.
You can take a test drive, and make suggestions.
```yaml
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/debian-v3.sh)"
```

**Comment by almostserious** [comment]
Oh thats awesome! 
Only thing I can currently think about that could be added. (Then again not not sure if this is required in the first setup) would be custom mountpoints.

i.e.: If I want an USB HDD to be accessible in a custom location in the LXC Container.
